src/ui/Filters/Filters.stories.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { storiesOf } from '@storybook/react';
+import Filters from 'ui/Filters/Filters';
+import styled from 'styled-components';
+
+const Page = styled.div`
+  background: ${props => props.theme.colors.linen.main};
+  height: 100vh;
+  padding: 2em;
+`;
+
+const initialFilters = {
+  format: [],
+  location: [],
+  topic: [],
+};
+
+const filterOptions = [{
+  filters: [{ _id: "0", name: "Raspberry" }, { _id: "1", name: "Vanilla" }, { _id: "2", name: "Sea Salt & Vinegar"}],
+  section: "topping",
+},{
+  filters: [{ _id: "3", name: "Blue" }, { _id: "4", name: "Golden" }, { _id: "5", name: "My Favorite Color"}],
+  section: "color",
+}];
+
+class FilterContainer extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeFilters: initialFilters,
+      filterOptions: filterOptions,
+    }
+  }
+
+  clearFilters = () => this.setState({ activeFilters: initialFilters});
+  setFilter = ({ accessor, filter }) => this.setState({ activeFilters: { ...this.state.activeFilters, [accessor]: filter }});
```

Comment:
We don't want to be creating a brand new component here in our stories... Stories should just accept stubbed out state/props and only render the component we're testing. Check out other stories to see how we simulate setting state (ie. Drawer.stories.js)